### DISCUSSION TITLE
Desk tool crashing when deployed.

### DISCUSSION THREAD

**Discussion by lourencelinao** [sanity-io/sanity#3909] [Sanity Studio Support]
Hello guys,

I have a NextJs App paired with Sanity in a monorepo

I am currently experiencing this issue, desk tool works fine locally but it crashes when I try to access it on a deployed environment.
Vision works but not the desk tool, has anyone experienced this before? Thanks in advance.

Sanity version: 3.0.0-rc.2
![image](https://user-images.githubusercontent.com/39908576/204301705-68feb322-8f2a-4985-9998-c2d1e902cadc.png)

**Comment by joaisa17** [comment]
Update your next.js version to the latest one. If that does not work, disable swcMinify in next.config.js.
